lib/formatters/__tests__/verboseFormatter.test.mjs

```diff
@@ -368,20 +368,19 @@ describe('verboseFormatter', () => {
 
 		expect(output).toBe(stripIndent`
 			file.css
-			 1:2  ×  Unexpected foo  no-foo
-			 1:2  ×  Unexpected bar  no-bar
-			 1:2  ×  Unexpected baz  no-baz
+			  1:2  ×  Unexpected foo  no-foo
+			  1:2  ×  Unexpected bar  no-bar
+			  1:2  ×  Unexpected baz  no-baz
 
-			3 problems (3 errors, 0 warnings)
+			× 3 problems (3 errors, 0 warnings)
+			  2 errors potentially fixable with the "--fix" option.
 
 			1 source checked
 			 file.css
```

Comment:
[suggest] The same indent level also in the file list seems a bit more readable. Any thoughts?

```suggestion
			  file.css
```

Reply:
Yes good idea. Updated 👍